### PR TITLE
Fixed indirect object notation

### PR DESCRIPTION
**Pull request by IvanLudvig**
Fixed 11 files so the program uses the preferred syntax instead of indirect object notation. Issue #73

### CODE REVIEW

File: bin/ar [hunk 1]
```diff
@@ -225,7 +225,7 @@ sub extractMember {
 				   oct($attr->[4]), int($attr->[1]));
     }
 
-    my $out = new FileHandle ">$name" or die "$name: $!\n";
+    my $out = FileHandle->new ">$name" or die "$name: $!\n";
```
Comment (briandfoy) [inline bin/ar]: Let's put some parens around the argument list (for this line and all the others like it)

FileHandle->new( ">$name" )

Comment (IvanLudvig) [reply bin/ar]: Added the parens

**Comment by briandfoy** [comment]
I'm tracking this, but I want to improve the tests to make sure everything happened correctly. That's just a limit on my time. I think the changes are good, but I also know that bugs don't care what I think. :)

**Comment by briandfoy** [comment]
I've merged this, but manually because I had to rebase onto previous pull requests. See 1302f660e0613e3733d3ec41fc90ec7c66fb08ee.

Thanks!

**Pull request closed without merging**